framework/include/timeintegrators/TimeIntegrator.h

```diff
@@ -127,6 +127,11 @@ class TimeIntegrator : public MooseObject,
    */
   virtual bool isExplicit() const { return false; }
 
+  /*
+   * Returns whether the solver controls it's own state
```

Comment:
```suggestion
   * Returns whether the solver controls its own state
```
is it the TI or the solver that controls the state?